packages/react-utilities/src/hooks/index.ts

```diff
@@ -1,16 +1,19 @@
-export * from './useControllableState';
 export * from './useBoolean';
 export * from './useConst';
+export * from './useControllableState';
 export * from './useControllableValue';
+export * from './useEvent';
 export * from './useEventCallback';
 export * from './useFirstMount';
+export * from './useForceUpdate';
 export * from './useId';
 export * from './useIsomorphicLayoutEffect';
 export * from './useMergedRefs';
 export * from './useMount';
 export * from './useOnClickOutside';
 export * from './useOnScrollOutside';
+export * from './usePointerCapture';
+export * from './usePointerCapture';
```

Comment:
These are duplicated
```suggestion
export * from './usePointerCapture';
```

Reply:
Oops that shouldn't be there